app/views/additional_payments/claims/nqt_in_academic_year_after_itt.html.erb

```diff
@@ -1,49 +1,56 @@
-<% content_for(:page_title, page_title(t("additional_payments.questions.nqt_in_academic_year_after_itt.heading"), journey: current_journey_routing_name, show_error: current_claim.errors.any?)) %>
+<% content_for(:page_title, page_title(t("additional_payments.questions.nqt_in_academic_year_after_itt.heading"), journey: current_journey_routing_name, show_error: @form.errors.any?)) %>
 <% path_for_form = current_claim.persisted? ? claim_path(current_journey_routing_name) : claims_path(current_journey_routing_name) %>
```

Comment:
There might be an obvious reason for `path_for_form` to exist, but I removed it from my form as I couldn't figure it out; if needed, it should be changed to call `#persisted?` on `@form`

Reply:
Yeah I'm not sure why that's there. With the form checking `persisted?` it should go to the right place 🤔 will remove it